tools/json-to-module/utils/bootstrap.js

```diff
@@ -84,7 +84,11 @@ if (!args.metadata) throw new Error("No metadata path specified");
 
     await Promise.all(
         newScenes.map(scene => {
-            return fs.writeJSON(`${args.metadata}/content/scene_info/${book}/${scene.name}.json`, scene, { spaces: 4 });
+            return fs.writeJSON(
+                `${args.metadata}/content/scene_info/${book}/${book}-${scene.flags.ddb.contentChunkId}-scene.json`,
```

Comment:
I saw some without a `contentChunkId`, this should do ` || "whatever-MrPrimate-used-for-tagging-scenes-without-a-chunk-id"` (and avoid conflicts if a file already exists with that name.)